src/rp2_common/hardware_pio/include/hardware/pio.h

```diff
@@ -382,6 +382,19 @@ static inline void sm_config_set_in_pins(pio_sm_config *c, uint in_base) {
     sm_config_set_in_pin_base(c, in_base);
 }
 
+/*! \brief Set the number of significant consecutive 'in' pins in a state machine configuration
```

Comment:
Should "significant" here be "least-significant" ?

Reply:
well i meant which are significant. perhaps better wording required